scripts/node

```diff
@@ -2,7 +2,7 @@
 
 exec docker run --rm \
     -u $(id -u):$(id -g) \
-    -e HOME=$PWD \
-    -v $PWD:$PWD:delegated \
```

Comment:
Are you sure "delegated" needs to be removed? It was just added in https://github.com/jenkins-infra/jenkins.io/commit/85720b5c6207c8bf52d6881b248aae5969a2184f